src/app/zap-templates/zcl/data-model/chip/general-commissioning-cluster.xml

```diff
@@ -79,7 +87,7 @@ limitations under the License.
         </mandatoryConform>
       </otherwiseConform>
     </attribute>
-    <attribute side="server" code="0x06" name="TCMinRequiredVersion" define="TC_MIN_REQUIRED_VERSION" type="int16u" writable="false" optional="true" apiMaturity="provisional">
+    <attribute side="server" code="0x06" name="TCMinRequiredVersion" define="TC_MIN_REQUIRED_VERSION" type="int16u" apiMaturity="provisional">
```

Comment:
So this is interesting.  The attribute is optional (if you look at the conformance), but the Python codegen no longer thinks it's optional?  Is the optionality not being propagated through ZAP correctly?

Reply:
Ah. we may need to upgrade ZAP to make this actually behave correctly...

Reply:
https://github.com/project-chip/connectedhomeip/pull/39132 for the ZAP update.

Reply:
Thanks for the ZAP update, yeah it seems that the new version was needed for this.